### DISCUSSION TITLE
Ability to search through sub-menus in the command line

### DISCUSSION THREAD

**Discussion by shenef** [monkeytypegame/monkeytype#2222] [Ideas] [answered]
Currently the command line only searches through the list of currently displayed options. 
It would be nice if searches from top level of the command line could show results from sub-menus.

Example: Entering the term "matrix" would result in the following results
matrix [Theme > matrix]
matrix [Keymap style > matrix]
split matrix [Keymap style > split matrix]
...

The text in square brackets could be justified to the right.
If you are already in a sub-menu then it's probably better to only search through the current menu since otherwise that clutters the list and can be confusing.

**Comment by Miodec** [comment, marked as answer]
Enable single list command line in the settings - it will allow you to search through the sub menus.
